linux_os/guide/system/accounts/accounts-session/accounts_tmout/bash/shared.sh

```diff
@@ -1,4 +1,4 @@
-# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_wrlinux
+# platform = multi_platform_rhel,Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_wrlinux
```

Comment:
Why not multi_platform_all here?

Reply:
Good point, the rule has an opt-in prodtype, so there is no reason to be too careful here.